src/features/ci-cd/hooks/service-installs.ts

```diff
@@ -54,6 +59,38 @@ const createReleaseServiceInstalls = async (
 	}
 	const serviceIds = services.map(({ id }) => id);
 
+	await handler(api, deviceIds, serviceIds);
+};
+
+const actOnAppServiceInstalls = async (
+	api: sbvrUtils.PinejsClient,
+	appIds: number[],
+	deviceIds: number[],
+	handler: (
+		api: sbvrUtils.PinejsClient,
+		deviceIds: number[],
+		serviceIds: number[],
+	) => Promise<void>,
+): Promise<void> =>
+	actOnReleaseServiceInstalls(
+		api,
+		deviceIds,
+		{
+			should_be_running_on__application: {
+				$any: {
+					$alias: 'a',
+					$expr: { a: { id: { $in: appIds } } },
```

Comment:
@Page- just wanted to verify that this is the correct way to do this.

Reply:
Yes this is valid/fine